api/v1/client/src/commonMain/kotlin/auth/TokenInfo.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2024 The ORT Server Authors (See <https://github.com/eclipse-apoapsis/ort-server/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.eclipse.apoapsis.ortserver.client.auth
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class TokenInfo(
+    /**
+     * The access token.
+     */
+    @SerialName("access_token")
```

Comment:
Instead of customizing each `@SerialName`, better provide a customized `SerialFormat` like

    val JSON = Json { namingStrategy = JsonNamingStrategy.SnakeCase }

and use that instance for serialization.

Reply:
That would require configuring this when creating the HttpClient for the AuthService. I’d prefer to keep the custom names directly in the data class.